.github/workflows/pythonpublish.yml

```
@@ -58,7 +58,7 @@ jobs:
           # https://docs.github.com/en/packages/learn-github-packages/publishing-a-package
           username: "${{ secrets.FLYTE_BOT_USERNAME }}"
           password: "${{ secrets.FLYTE_BOT_PAT }}"
-          image_name: ${{ github.repository_owner }}/flytekit
+          image_name: ${{ github.repository_owner }}/flytekit-py37
           image_tag: flytekit-latest,flytekit-${{ github.sha }},flytekit-${{ steps.bump.outputs.version }}
```

Something like this:
```suggestion
          image_tag: py37-latest,py37-${{ github.sha }},py37-${{ steps.bump.outputs.version }}
```

---

ok